ovos_dinkum_listener/voice_loop/voice_loop.py

```diff
@@ -650,6 +653,15 @@ def _after_cmd(self, chunk: bytes):
         # Command has ended, call transformers pipeline before STT
         chunk, stt_context = self.transformers.transform(chunk)
 
+        if isinstance(self.stt, FakeStreamingSTT) and self.remove_silence:
+            # NOTE: This is using the FS-STT buffer directly, not the S-STT queue
+            self.stt.stream.buffer.clear()
+            extracted_speech = self.vad.extract_speech(self.stt_audio_bytes)
```

Comment:
need to require minimum OPM version where this was introduced in requirements.txt